### ISSUE TITLE
Can consumers provide annotation configuration?

### ISSUE THREAD

**Issue by wangweifan** [apache/rocketmq#482]
for example 
@RabbitListener(queues = "ccs.queue.sync")
public void hello(String id, @Header("jobName") String jobName) {
        System.out.println("Received request for id " + id);
        System.out.println("Received request for job name " + jobName);
}

**Comment by walking98** [comment]
@wangweifan yes, please try this spring-boot function provided in rocketmq-external. https://github.com/apache/rocketmq-externals/tree/master/rocketmq-spring-boot-starter.
